README.md

```diff
@@ -94,3 +94,6 @@ Probably the best thing you can do to get a feature promoted is to draft an RFC
 This repository is governed by the [Contributor Covenant Code of Conduct](https://www.contributor-covenant.org/).
 
 We pledge to be overt in our openness, welcoming all people to contribute, and pledging in return to value them as whole human beings and to foster an atmosphere of kindness, cooperation, and understanding.
+
+We additionally request that the use of personal pronouns ("you", referring to a specific person) not generally be used in discussions - obviously there are exceptions, especially in thanking or other kindnesses, but an effort should be made to make technical points in a non-personal way. If a personalized mode is used for technical argument the moderators may delete these contributions, requesting that content be reposted without making arguments in this way.
```

Comment:
The PR description suggests that the intention is only to restrict the word "you". This sentence asserts that the only personal pronoun is "you" which just appears incorrect. It is then left unclear whether other personal pronouns such as "I" and "it" are banned. Banning "it" in particular causes the well known problems documented in Monty Python and the Holy Grail.

It is likely that this PR is an attempt is to avoid ad-hominem arguments, and another enumeration that seems incorrect is the list of things that are not ad-hominem arguments (thanking, kindnesses). This would need to be enlarged (requests, clarifications, ...) to include everything that is not an ad-hominem argument. Assuming that this is the intention, I would recommend banning ad-hominem arguments instead of banning everything and then making individual exceptions for everything that is not an ad-hominem.

Reply:
I'll clarify, thanks.

Reply:
Clarified here https://github.com/fsharp/fslang-suggestions/blob/master/README.md#code-of-conduct

Reply:
Restricting use of particular word(s) sounds odd to me, however I think I understand the goal. Is the main goal to avoid ad hominems / personal attacks? If so, it might be better stated that way, and avoids having to stop and think every time before using pronouns in places where it's really not a problem.
Case in point -- the word "me" appears twice in this comment and "I" twice, but I don't think this comment is particularly fallacious.

Reply:
This specific technique - "minimize the use of the word 'you', meaning a specific person" - is one I learned in 2008 during the design of F# 2.0, and have tried to apply throughout, so I guess it's part of the spirit of the F# design process, and I'm happy with requesting it.

It's true it helps prevent ad-hominem negative comments, but long before they ever come close to being negative. And I've found it a very accurate way of assessing whether conversations are in the early stages of being on that kind of trajectory.  Simply asking someone to use "you" a bit less is a great way of dialling it down, while still keeping the door open to productive conversations.

> avoids having to stop and think every time before using pronouns in places where it's really not a problem.

It's only specifically "you" or "your". And only when it means a specific person. And obviously negative references are the problematic ones to look for.  It's not a hard and fast rule, but if a message uses "you" or "you" 50 times then consider taking a walk and redrafting it :)

Reply:
_This comment is partially machine-translated from German to English_

Even though this is already merged: I like the idea of ​​having a set of simple rules that almost automatically lead to the communication style that shall be maintained here. Two things come to mind:

* The intention of the rules should be clear and understandable for every person. The opening sentence of the "Code of Conduct" already points in this direction, but is kept very general. Perhaps a description of the desired style of discussion would be helpful (even if this should essentially be "common-sense" in a civilized society; e.g. mutual consideration / allowing oneself the possibility of error / assuming misunderstanding rather than malice / etc.).

* I find the rule listed here interesting and good because it is very easy to use and leads to something - as if by itself. There are several such simple, effective rules. How about adding more rules? I can think of these that have helped me a lot personally:
	* "If you feel like being in a huff - especially if you don't now why - take an hour of break before you act". And I'm serious here: This is my favorite Corona-We-All-Stay-At-Home-Together rule, and it works. There are for sure better formulations. I like this rule in discussions, because they lead to calm down the heat of a discussion or to stay cool.
	* "Avoid quoted expressions (they leave (too?) much room for the reader's interpretation). Instead, make an effort to clarifying what you mean".
	* "Sometimes it is helpful to pack your thoughts in a question, instead of making a statement".
	* I'm not a communication expret, but I guess there are many more. The "you" rule caught my interest, and I already applied it in my work the last 2 days - and it just works! Thanks for that!


I also notice:

* The wording chosen here (especially the 1st sentence of the "Code of Conduct") could be difficult or at least not immediately understandable for people whose mother tongue is not English. Of course, there is always the risk of distorting the content by simplifying wording - but it should be taken into account that there is an international community who are not native English speakers; in this Readme, and also in other places of this repo and related repos (I notice this from time to time).